apps/design/frontend/src/image_input.tsx

```diff
@@ -177,6 +177,19 @@ export function ImageInput({
       <ImageInputButton disabled={disabled} onChange={onChange}>
         {buttonLabel}
       </ImageInputButton>
+      {value && (
+        <Button
+          onPress={() => onChange(undefined)}
+          disabled={disabled}
+          variant="danger"
+          icon="Delete"
+          style={{
+            marginLeft: '0.5rem',
+          }}
+        >
+          Remove Image
```

Comment:
I think this text should be a prop, similar to `buttonLabel`, so they can be consistent with each other. Or we standardize them to Upload Image and Remove Image

Reply:
96e4d3d8dd